aspnetcore/security/enforcing-ssl.md

```diff
@@ -113,6 +129,13 @@ Per [OWASP](https://www.owasp.org/index.php/About_The_Open_Web_Application_Secur
 * The browser stores configuration for the domain that prevents sending any communication over HTTP. The browser forces all communication over HTTPS. 
 * The browser prevents the user from using untrusted or invalid certificates. The browser disables prompts that allow a user to temporarily trust such a certificate.
 
+HSTS is a client-side performance optimization to avoid an extra request to the server. HSTS limitations:
+
+* The client needing to support HSTS.
+* HSTS is not guaranteed to limit traffic to HTTPS only.
+* HSTS requires at least one successful HTTPS request to establish the HSTS policy.
+* Application must check every HTTP request and redirect, reject, etc.
```

Comment:
* "The app ..."
* "etc." banned ... and I don't follow : What is an "HTTP reject"? How does an app check a "reject"?